test/conftest.py

```diff
@@ -11,6 +11,8 @@
 from psutil import virtual_memory
```

Comment:
I'm afraid that the current .gitlab-ci.yml doesn't run the tests. At least that's what the log says on line 119:

https://gitlab.ebi.ac.uk/EGA/ega-download-client/-/jobs/494312#L119

Reply:
Running `pytest` does run the tests, but at least one of them fails.

Reply:
I decided that the test is not valuable because just checking that it exists it's not useful for us to know. I've we want a test like this then we should write tests to try different arguments and check the output properly.